modules/unsupported/swing/src/main/java/org/geotools/swing/data/JParameterListWizard.java

```diff
@@ -33,11 +33,11 @@
  * <p>Example of use (from the GeoTools example project):
  *
  * <pre><code>
- * List<Parameter<?>> list = new ArrayList<Parameter<?>>();
- * list.add(new Parameter<File>("image", File.class, "Image",
+ * List&lt;Parameter&lt;?&gt;&gt; list = new ArrayList&lt;Parameter&lt;?&gt;&gt;();
+ * list.add(new Parameter&lt;File&gt;("image", File.class, "Image",
  * "GeoTiff or World+Image to display as basemap",
  * new KVP( Parameter.EXT, "tif", Parameter.EXT, "jpg")));
- * list.add(new Parameter<File>("shape", File.class, "Shapefile",
+ * list.add(new Parameter&lt;File&gt;("shape", File.class, "Shapefile",
  * "Shapefile contents to display", new KVP(Parameter.EXT, "shp")));
```

Comment:
this is just a hint; 
you could use the `{@code ...}` instead of all the HTML entities, it leads to much more readable comments in the java code.
I think the `<pre>` (preformatted text) tag/block may still be needed to preserve line breaks; not sure about that as it is not a valid HTML5 tag and javadoc (11 and up generate HTML5)

see eg. https://stackoverflow.com/questions/541920/multiple-line-code-example-in-javadoc-comment

Reply:
Because there is so much of this, I'm doing the minimal amount of changes to clean up the warnings. No problem with additional fixes later, or additional suggestions.